### ISSUE TITLE
Accessibility issue with Grid component: screen readers cannot navigate up/down columns

### ISSUE THREAD

**Issue by minorninth** [bvaughn/react-virtualized#1878]
## Bug Report

1. On macOS, enable VoiceOver and visit this page:
https://bvaughn.github.io/react-virtualized/#/components/Grid

2. Navigate to the grid and interact in
3. Try to use VoiceOver+Up/Down to move up/down in the table

### What is the current behavior?

VoiceOver moves to the next/previous cell within the row

### What is the expected behavior?

VoiceOver should be able to navigate two-dimensionally.

### Which versions of React and react-virtualized, and which browser / OS are affected by this issue? Did this work in previous versions of react-virtualized?

Tested on macOS 15.4 with Safari, but based on the markup this is likely an issue with nearly any browser and any screen reader. All screen readers have the ability to navigate two-dimensionally in HTML tables and grids, but the markup that the react-virtualized Grid component is generating puts everything in one row.
